bin/cleanup/cleanup.py

```diff
@@ -413,6 +416,53 @@ def cleanup_client(
         raise
 
 
+# cleanup_gamma_server creates a server runner, and calls its cleanup() method.
+def cleanup_gamma_server(
+    project,
+    network,
+    k8s_api_manager,
+    server_namespace,
+    gcp_api_manager,
+    gcp_service_account,
+    *,
+    suffix: Optional[str] = "",
+):
+    deployment_name = xds_flags.SERVER_NAME.value
+    if suffix:
+        deployment_name = f"{deployment_name}-{suffix}"
+
+    ns = k8s.KubernetesNamespace(k8s_api_manager, server_namespace)
+    # Shorten the timeout to avoid waiting for the stuck namespaces.
+    # Normal ns deletion during the cleanup takes less two minutes.
+    ns.wait_for_namespace_deleted_timeout_sec = 5 * 60
+    server_runner = _GammaServerRunner(
+        k8s_namespace=ns,
+        deployment_name=deployment_name,
+        gcp_project=project,
+        network=network,
+        gcp_service_account=gcp_service_account,
+        gcp_api_manager=gcp_api_manager,
```

Comment:
We need to pass `enable_workload_identity` from `xds_k8s_flags.ENABLE_WORKLOAD_IDENTITY.value`. This flag is true by default, so it's not a big deal it's omitted in the cleanup_server, but in gamma tests it's configured to be disabled by default in the flags.